pages/linux/strace.md

```diff
@@ -0,0 +1,16 @@
+# strace 
+
+> Troubleshooting tool for tracing system calls
+
+- Start tracing a specific PID 
```

Comment:
Perhaps "a specific process by its PID" would be clearer, and more consistent with the phrasing of the examples below.